docs/04-Interactive-API.md

```
@@ -1537,6 +1537,343 @@ DELETE /api/ui/interactive/v2/data?id={windowId}
+    };
+    var content = new StringContent(payload.ToString(), Encoding.UTF8, "application/json");
+    var clickResp = await http.PostAsync($"{uiUrl}/api/ui/interactive/v2/tools", content);
+    clickResp.EnsureSuccessStatusCode();
+
+    return JObject.Parse(await clickResp.Content.ReadAsStringAsync());
+}
+```
+
+<!-- /tabs -->
+
+> **Note:** The `GET /tools` endpoint uses `?windowId=` (not `?id=`). See the [query parameter inconsistency note](#data-operations-v2-recommended) in the Endpoints section.
```

markdownlint-cli2 (0.22.0)

[warning] Link fragments should be valid

(MD051, link-fragments)